Source/WebCore/rendering/RenderBox.cpp

```diff
@@ -5211,30 +5211,51 @@ void RenderBox::addOverflowFromChild(const RenderBox& child, const LayoutSize& d
     addVisualOverflow(*childVisualOverflowRect);
 }
 
+LayoutBoxExtent RenderBox::allowedLayoutOverflow() const
+{
+    LayoutBoxExtent allowance { -1, -1, -1, -1 };
```

Comment:
We try to avoid magic numbers like -1, preferring `std::optional<>` instead. Maybe this should be a `RectEdges<std::optional<LayoutUnit>>` ?

Reply:
Done, and added LayoutOptionalOutsets as an alias.